udata/migrations/2021-08-17-oauth2client-integrity.py

```diff
@@ -0,0 +1,27 @@
+'''
+Remove Oauth2Client db integrity problems
+'''
+import logging
+
+import mongoengine
+
+from udata.api.oauth2 import OAuth2Client
+
+log = logging.getLogger(__name__)
+
+
+def migrate(db):
+    log.info('Processing Transfer objects.')
+
+    clients = OAuth2Client.objects.no_cache().all()
+
+    count = 0
+    for client in clients:
+        try:
+            client.organization and client.organization.id
```

Comment:
Can't we directly use `client.organization.id`?

Reply:
Nope because sometimes it's unset (not required).